package.json

```diff
@@ -22,9 +22,9 @@
   },
   "devDependencies": {
     "LiveScript": "latest",
-    "typescript-compiler": "^1.4",
+    "accord-babel-legacy": "^1.0.0",
     "acorn": "latest",
-    "babel": "latest",
+    "babel": "^5.8.38",
```

Comment:
Are you sure about this? Works fine for me and the tests pass on my machine. The normal Babel adapter seems to only use `babel-core` and the legacy babel adapter uses the `babel` module.